packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -61,6 +61,9 @@ export function SignIn() {
             className="amplify-flex"
             disabled={isPending}
           >
+            <VisuallyHidden>
+              <legend>Sign in</legend>
```

Comment:
Please split this change out into a separate PR and include the Vue and Angular SignIn screens as well. I'd like this to be reviewed in a non-docs change PR since it's more about Authenticator accessibility than strictly the docs.